py3status/modules/deadbeef.py

```diff
@@ -67,56 +79,48 @@ def _is_running(self):
 
     def deadbeef(self):
         color = self.color_stopped
-        artist = isplaying = length = playback_time = title = \
-            tracknumber = year = ""
+        status = ''
```

Comment:
The original issue was that I left literal `return out` or something like that right before the py3 return. Will try `None`.

Are you okay with using `self.plist` as a shortened `self.placeholders`?